unison-cli/src/Unison/Codebase/Editor/Input.hs

```diff
@@ -1,36 +1,38 @@
 {- ORMOLU_DISABLE -} -- Remove this when the file is ready to be auto-formatted
 module Unison.Codebase.Editor.Input
-  ( Input(..)
-  , GistInput(..)
-  , Event(..)
-  , OutputLocation(..)
-  , PatchPath
-  , BranchId, AbsBranchId, parseBranchId
-  , HashOrHQSplit'
-  , Insistence(..)
-  ) where
-
-import Unison.Prelude
+  ( Input (..),
```

Comment:
I ran ormolu over the imports.

Reply:
If you do that, go ahead and move the `{- ORMOLU_DISABLE -}` below the imports for the future.

Reply:
I've tried that and it doesn't work on my machine, but if it works on yours I can do that.

Reply:
Oh, no, I haven't tried it. Do you mean that when you tried moving `{- ORMOLU_DISABLE -}` below the imports, ormolu still didn't format the imports? Or what didn't work?

Reply:
> Do you mean that when you tried moving {- ORMOLU_DISABLE -} below the imports, ormolu still didn't format the imports?

Yes, that's what I observed, but didn't dig into why that's the case (it didn't seem super important)

Reply:
Ok, I opened #2746 to track.